codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/query/customization.config

```diff
@@ -21,5 +21,11 @@
         }
       }
     }
-  ]
+  ],
+  "customRequestTransformerMap": {
```

Comment:
**nit**: I would drop the `map` suffix (everything is a map in json), simply `customRequestTransformers` sounds more clear to me. Other keys, like `customOperationContextParams` don't have the map suffix, it would make the customization file more consistent. But it is not a blocker for me.